Dockerfile

```diff
@@ -22,10 +22,12 @@ SHELL ["/bin/bash", "-c"]
 WORKDIR /usr/local/src/security_monkey
 COPY requirements.txt /usr/local/src/security_monkey/
 
+RUN echo "UTC" > /etc/timezone
+
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install --no-install-recommends -y build-essential python-pip python-dev && \
-    apt-get install --no-install-recommends -y wget postgresql postgresql-contrib libpq-dev libffi-dev libxml2-dev libxmlsec1-dev && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y build-essential python-pip python-dev && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y wget postgresql postgresql-contrib libpq-dev libffi-dev libxml2-dev libxmlsec1-dev && \
```

Comment:
DEBIAN_FRONTEND=noninteractive && ?
maybe just set DEBIAN_FRONTEND as an `ENV DEBIAN_FRONTEND=noninteractive`

Reply:
This comment says we shouldn't do that: https://github.com/phusion/baseimage-docker/issues/58#issuecomment-206354591.

Reply:
Without "&&" it works.